### DISCUSSION TITLE
bug with the display of the site on a mobile phone

### DISCUSSION THREAD

**Discussion by ignavan39** [community/community#49169] [Mobile]
### Select Topic Area

Bug

### Body

the site is displayed as if it is a desktop version, only on user pages
![Screenshot_2023-03-04-19-37-27-988_com android chrome](https://user-images.githubusercontent.com/68580920/222905769-0def73cf-7590-450c-a441-c0a12bb6f079.jpg)

I'm disabled "desktop site"

Android
Google chrome

**Comment by ettaboyle** [comment]
Welcome to the GitHub Community, @ignavan39, we're happy you're here! 

You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you from `general` to `mobile`. Good luck!

**Reply by jsoref** [reply]
Isn't Mobile for the app and not the web site?

This feels like something better handled by https://support.github.com
